packages/expo-document-picker/src/ExpoDocumentPicker.web.ts

```diff
@@ -59,7 +59,7 @@ function readFileAsync(targetFile: File, base64: boolean = true): Promise<Docume
     const mimeType = targetFile.type;
     if (!base64) {
       resolve({
-        uri: targetFile.webkitRelativePath,
+        uri: URL.createObjectURL(targetFile),
```

Comment:
This now returns a Blob by default, which is fast and directly usable. Before this change, users never got a workable URI on web unless they manually created an object URL from the file.